webpack.config.js

```diff
@@ -6,6 +6,7 @@ var index = require('./lib/index-static.jsx');
 
 module.exports = {
   entry: './lib/main.jsx',
+  devtool: 'source-map',
```

Comment:
Exactly the problem I was facing and had to set to `eval` to get this working both on debugger and console, but lets land this one for now and document that in order to get the console to work they either have to set `eval` manually for Firefox or use the debugger.